dvc/command/commit.py

```diff
@@ -44,18 +44,18 @@ def add_parser(subparsers, parent_parser):
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
     commit_parser.add_argument(
-        "-f",
-        "--force",
+        "-d",
```

Comment:
Let's preserve the old order, so that `-d` is closer to `-R`, since they are related. Probably worth changing the other command instead. Also, have you checked other commands that have these arguments?

Reply:
Actually, I was thinking to make them in alphabetical order. In `checkout` (other than commit), it's `-d` and then `-f`. Other commands don't have all the three options. So, what order should I follow finally?

Reply:
I don't think in this case alphabetic order is important. Keep the related options close together is more important. Like [-q, -v, -h] - auxiliary options, [-d, -R] is the way to limit the scope (along with providing targets),  [-a, -T] - a way to expand and check other workspaces, etc. So, in this particular case let's keep them together. Check pull, push, repro, fetch, gc and make sure they are together as well. Check if there are other commands with -R.

Reply:
Done, PTAL! I checked these files but none of them has both -d and -R commands.

Reply:
@vibhor98 awesome! thanks. Please check this one https://dvc.org/doc/commands-reference/pull - it has -R and -d and -f between them. Please also, update the docs - synopsis and the option description order.

Reply:
Actually I checked it but in `pull.py` file, there are no commands defined whose order I can change. Can you please help me where they are defined? Also, I'll update the docs by making PR in dvc.org repo.

Reply:
Done, thanks!